docker-compose.yml

```diff
@@ -4,3 +4,5 @@ services:
     container_name: fren-bot
     build: .
     restart: unless-stopped
+    volumes:
+      - ./.env.production.local:/usr/src/app/.env.production.local
```

Comment:
Does this fail if that file doesn't exist?

Reply:
probably - but the bot doesn't work without the tokens, so i think that's fine

Reply:
also this is on the wrong commit - my bad. this should go in #29